.github/command-docs.md

```diff
@@ -79,7 +79,7 @@
 | .msfs           | Provides links to MSFS support for sim issues                                     | .msfsforum                                         |
 | .navdata        | Explains the use of navdata by the FlyByWire A32NX and how to check its accuracy  | ---                                                |
 | .navigraphdata  | Provides help with Navigraph NavData reinstall                                    | .navigraphnavdata                                  |
-| .reportedissues | Provides a link to the reported issues page within docs                           | .issues                                            |
+| .reportedissues | Provides a link to the reported issues page within docs, either a general link, or a link for a specific search <br> The search term "autopilot" leads to the autopilot typical issues page <br> Search terms can be terminated by a dot, line break, or greather-than sign                     | .issues                                            |
```

Comment:
Can you fix table alignment here please? It doesn't affect anything but would look neater (Depending on your IDE - it may have a tool to do it for you)

Reply:
Aligning the table makes all rows be longer than the screen width, so the whole table isn't really readable in code anymore, so I would rather leave that one row as is.
Formatted table:
![Screenshot 2023-03-12 114703](https://user-images.githubusercontent.com/79196358/224542419-bcd17534-8f6c-4732-bb32-8ef59d8c6aa4.png)
Line not aligned:
![Screenshot 2023-03-12 115413](https://user-images.githubusercontent.com/79196358/224542422-9f468edf-c9d0-49b0-ad59-cdefe09def1d.png)